OpenSSL/crypto.py

```diff
@@ -1008,11 +1098,14 @@ def sign(self, pkey, digest):
 
     def get_signature_algorithm(self):
         """
-        Retrieve the signature algorithm used in the certificate
+        Return the signature algorithm used in the certificate.
+
+        :return: The name of the algorithm.
+        :rtype: :py:class:`bytes`
+
+        :raises ValueError: If the signature algorithm is undefined.
 
-        :return: A byte string giving the name of the signature algorithm used in
-                 the certificate.
-        :raise ValueError: If the signature algorithm is undefined.
+        ..versionadded:: 0.13
```

Comment:
This doesn’t render properly because it lacks a space btw `..` and `versionadded`.